### PR TITLE
Added dotnet core 2.2 as Targeting framework

### PR DESCRIPTION
**Pull request by prashantchoudhary**
Hi,
I have added dotnet core 2.2 as Targeting framework as it is the most popular framework version currently. I didn't find any breakages so far.

### CODE REVIEW

File: src/stream-net-tests/stream-net-tests.csproj [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
-    <PackageReference Include="NUnit" Version="3.9.0" />
-    <PackageReference Include="NUnit.Console" Version="3.7.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.9.0" />
-    <PackageReference Include="OpenCover" Version="4.6.519" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
+    <PackageReference Include="NUnit.Console" Version="3.10.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
+    <PackageReference Include="OpenCover" Version="4.7.922" />
```
Comment (prashantchoudhary) [inline src/stream-net-tests/stream-net-tests.csproj]: Migrated to latest versions of nuget packages.

File: src/stream-net/stream-net.csproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <DefineConstants>TRACE;DEBUG;NETCORE</DefineConstants>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>net45;net461;netcoreapp1.0;netstandard1.6;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net45;net461;netcoreapp1.0;netcoreapp2.2;netstandard1.6;netstandard2.0</TargetFrameworks>
```
Comment (prashantchoudhary) [inline src/stream-net/stream-net.csproj]: Added dot net core 2.2 as targeted framework

File: src/stream-net/stream-net.csproj [hunk 3]
```diff
@@ -22,7 +22,7 @@
     <DefineConstants>NET45</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```
Comment (prashantchoudhary) [inline src/stream-net/stream-net.csproj]: upgraded Newtonssoft json

**Comment by prashantchoudhary** [comment]
Test cases are passing locally. Not sure why it is failing here.

**Comment by prashantchoudhary** [comment]
Succeeded after retrying.

File: src/stream-net/stream-net.csproj (hunk 2 above)
Comment (glitch100) [reply src/stream-net/stream-net.csproj:6]: Should add netcore3.1 too

**Comment by glitch100** [comment]
Once this is merged I have some PR's to go up

File: src/stream-net/stream-net.csproj (hunk 2 above)
Comment (prashantchoudhary) [reply src/stream-net/stream-net.csproj:6]: sure will add, possibly 5.0 too

Comment (ferhatelmas) [reply src/stream-net/stream-net.csproj:6]: That sounds great :+1: 

We can bump the major and kill old stuff then.

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Review by imbird — COMMENTED** [review]
THIS INTEGRATION TEST WILL FAIL. Massive problems. For details, inquire within.

**Comment by imbird** [comment]
Also, hey, what are YOU doing in here? 

It's quite telling that I literally thought this was a part of the _same project_ as I'm seeing all of the related items here about... 

Kindly note that THAT project is the OPPOSITE of open source...

**Pull request merged**